composer.lock

```diff
@@ -1296,33 +1296,37 @@
         },
         {
             "name": "guzzlehttp/psr7",
-            "version": "1.5.2",
+            "version": "1.6.1",
```

Comment:
Let's try to keep it in mind and talk about it after the next daily.

Reply:
The `composer.lock` for the FundraisingFrontend is essential to have stable deployments and avoid accidentally deploying wrong master of any bounded context dependencies. If the frequent updates to the messages cause merge conflicts (which are easily resolved) that bother us too much, then we should rather think of a different way of incorporating the messages into the dev environment.

Reply:
The thing is we never commit a `composer.lock` with only the content repo version bumped up. It's always a few things, because it's kind of annoying having to run update and then hand pick only the content repo in the lock file. And all of those random composer updates people run locally tend to collide with each other now and then. I didn't say the merge conflicts are hard to resolve but it'd be nice if we don't have them so often.
I admit  I don't understand the argument that we use the lock file to avoid accidentally deploying the wrong master, so let's discuss it in the next daily.